Clients/Xamarin.Interactive.Client.Windows/ViewModels/InspectTreeNode3d.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System;
```

Comment:
Missing header? Also separate namespace groups.

Reply:
Name of the file should be `InspectTreeNode3D` not `InspectTreeNode3d`